stress-ng.c

```diff
@@ -1703,6 +1739,7 @@ static void MLOCKED_TEXT stress_run(
 						(void)stress_perf_enable(&stats->sp);
 #endif
 					if (keep_stressing_flag() && !(g_opt_flags & OPT_FLAGS_DRY_RUN)) {
+
```

Comment:
I'd prefer the empty line to be removed. Thanks!

Reply:
Sure, will update in v2.